test/parallel/test-process-kill-pid.js

```diff
@@ -47,7 +47,7 @@ function kill(tryPid, trySig, expectPid, expectSig) {
 
   process.kill(tryPid, trySig);
 
-  assert.strictEqual(getPid, expectPid);
+  assert.strictEqual(getPid.toString(), expectPid.toString());
```

Comment:
This is run several times, and at various times `getPid` can be a number or a string (`expectPid` is always a number). I could also do: 
```js
assert.strictEqual(Number.parseInt(getPid), expectPid)
```

if you think that's better.